tools/engine_tool/lib/main.dart

```diff
@@ -66,13 +66,13 @@ void main(List<String> args) async {
     platform: const LocalPlatform(),
     processRunner: ProcessRunner(),
     logger: Logger(),
+    verbose: verbose,
```

Comment:
Optional: as a further cleanup, the verbosity could just be encoded in the log level of the `Logger()` on the line above. It's currently set in https://github.com/flutter/engine/blob/main/tools/engine_tool/lib/src/commands/command_runner.dart#L106, but could just as well be set here.

Reply:
I think this makes sense, but probably what I would do is something like `Logger(verbose: true)` and have Logger set itself up versus trying to overload Logger's log severity (where verbose isn't really a thing) with the app-wide verbosity flag.

Let me send a follow-up PR.